pkgs/os-specific/linux/libnvme/default.nix

```
@@ -0,0 +1,62 @@
+{
+  fetchFromGitHub,
+  json_c,
+  lib,
+  libuuid,
+  meson,
+  ninja,
+  openssl,
+  perl,
+  pkg-config,
+  python3,
+  stdenv,
+  systemd,
+}:
+stdenv.mkDerivation rec {
+  pname = "libnvme";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "linux-nvme";
+    repo = "libnvme";
+    rev = "v${version}";
+    sha256 = "EPAPWY6/Bh8I1eLslKJAofLn0IAizmGn00Q5PJPtdRw=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    perl # for kernel-doc
+    pkg-config
+  ];
+
+  buildInputs = [
+    json_c
+    libuuid
+    openssl
+    python3
+    systemd
+  ];
+
+  mesonFlags = [
+    "-Ddocs=man"
+    "-Ddocs-build=true"
+  ];
+
+  postPatch = ''
+    chmod +x doc/kernel-doc-check
+    patchShebangs .
+  '';
+
+  doCheck = true;
+
+  outputs = ["out" "man"];
+
+  meta = {
```

Please do this for `meta`.
```suggestion
  meta = with lib; {
```